src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const keyToRemove of action.keysToRemove) {
+          delete state[keyToRemove];
+        }
+        break;
+
+      case 'clear':
+        for (const key in state) {
```

Comment:
Keep one way of naming. Here should be `keyToRemove` too

Reply:
do I really need it here? I'm using keyToRemove with for...of loop action.keysToRemove

Reply:
same meaning